common/templates/partials/offenderDetails.njk

```diff
@@ -1,4 +1,4 @@
-{% if offenderDetails %}
+{% if offenderDetails and not hideOffenderDetails %}
```

Comment:
we need this as the success page needs the offenderDetails but the design doesn't include the offenderDetails header